snapshots/overlay/overlayutils/check.go

```diff
@@ -168,3 +175,15 @@ func NeedsUserXAttr(d string) (bool, error) {
 	}
 	return true, nil
 }
+
+// figure out whether "index=off" option is recognized by the kernel
```

Comment:
This should use the standard format for GoDoc (starting with the function name), e.g. something like;

```suggestion
// SupportsIndex checks whether the "index=off" option is supported by the kernel.
```

Reply:
done thank you